workers/unity/Packages/io.improbable.gdk.debug/NetStatsViewer/NetStatViewer.cs

```diff
@@ -53,17 +63,41 @@ private void SetupUI()
             var windowTemplate = AssetDatabase.LoadAssetAtPath<VisualTreeAsset>(WindowUxmlPath);
             windowTemplate.CloneTree(rootVisualElement);
 
-            // Load update row
-            var itemTemplate = AssetDatabase.LoadAssetAtPath<VisualTreeAsset>(UpdateRowUxmlPath);
+            // Initialize tabs
+            var updatesTab = rootVisualElement.Q<NetStatsUpdatesTab>();
+            updatesTab.InitializeTab();
+
+            var commandsTab = rootVisualElement.Q<NetStatsCommandsTab>();
+            commandsTab.InitializeTab();
+
+            var worldCommandsTab = rootVisualElement.Q<NetStatsWorldCommandsTab>();
+            worldCommandsTab.InitializeTab();
+
+            // Setup tab buttons
+            tabs = new Dictionary<Tab, NetStatsTab>
+            {
+                { Tab.Updates, updatesTab },
+                { Tab.Commands, commandsTab },
+                { Tab.WorldCommands, worldCommandsTab }
+            };
+
+            rootVisualElement.Q<ToolbarButton>("updateSelector").clickable.clicked += () => SelectTab(Tab.Updates);
+            rootVisualElement.Q<ToolbarButton>("commandSelector").clickable.clicked += () => SelectTab(Tab.Commands);
+            rootVisualElement.Q<ToolbarButton>("worldCommandSelector").clickable.clicked +=
+                () => SelectTab(Tab.WorldCommands);
+        }
 
-            // Clone row for each component, fill list
-            var scrollView = rootVisualElement.Q<ScrollView>("updatesContainer");
-            for (var i = 0; i < spatialComponents.Count; i++)
+        private void SelectTab(Tab tabType)
+        {
+            foreach (var pair in tabs)
             {
-                var element = itemTemplate.CloneTree();
-                UpdateElement(i, element);
-                scrollView.Add(element);
+                pair.Value.AddToClassList("tab-hidden");
```

Comment:
Does this ensure that these are unique? I.e. - if `tab-hidden` already exists, it will not add it again

Reply:
Yes, this is the exact method the UI builder uses for hiding and showing items as well